components/Choice/index.jsx

```diff
@@ -21,6 +21,11 @@ export default class Choice extends React.Component {
     heading: PropTypes.string,
     headingStyle: PropTypes.string,
     isRequired: PropTypes.bool,
+    label: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.object
+    ]),
+    labelHidden: PropTypes.bool,
```

Comment:
I think a lot of the props are pretty confusing right now. I would rather wait till we have a proper docs page (such as react-storybook) to explain each prop rather than adding tons of comments in each component.